### PR TITLE
Update to WordPress version 4.9.

### PR DESCRIPTION
**Pull request by greg-1-anderson**
Inspect the result of the [functional tests run by Circle CI](https://circleci.com/gh/pantheon-systems/WordPress).

**OPTIONAL** -- To create your own test site:

- Create a new WordPress site on Pantheon.
- When site creation is finished, visit dashboard.
- Switch to "git" mode.
- Clone your site locally.
- Apply the files from this PR on top of your local checkout.
  - git remote add WordPress git@github.com:pantheon-systems/WordPress.git
  - git fetch WordPress
  - git merge WordPress/update-4.9
- Push your files back up to Pantheon.
- Switch back to sftp mode.
- Visit your site and step through the installation process.

### CODE REVIEW

File: wp-includes/version.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
  *
  * @global string $wp_version
  */
-$wp_version = '4.8.2';
+$wp_version = '4.8.3';
```
Comment (ataylorme) [inline wp-includes/version.php]: @greg-1-anderson this looks like an update to `4.8.3` not `4.9`

Comment (greg-1-anderson) [reply wp-includes/version.php]: Drat.

Comment (greg-1-anderson) [reply wp-includes/version.php]: I've got 4.9 on the Pantheon update site...

Comment (greg-1-anderson) [reply wp-includes/version.php]:
```
terminus env:commit wp-update.update --yes '--message=Update to WordPress 4.9. For more information, see '
 [warning] There is no code to commit.
```
☹️

**Comment by greg-1-anderson** [comment]
@ataylorme OK, Terminus was confused about whether or not there were commits available. I think it may have been caused by a Dashboard bug.

This PR should now have WordPress 4.9 on it, methinks.

**Comment by ataylorme** [comment]
👍

**Comment by greg-1-anderson** [comment]
This has been deployed.

**Pull request closed without merging**